test/importer/MusicXmlImporterTestSuite.test.ts

```diff
@@ -606,4 +606,12 @@ describe('MusicXmlImporterTestSuiteTests', () => {
             'test-data/musicxml-testsuite/99b-Lyrics-BeamsMelismata-IgnoreBeams.xml'
         );
     });
+
+    it('100a_Guitare_Bends', async () => {
+        await MusicXmlImporterTestHelper.testReferenceFile(
+            'test-data/musicxml-testsuite/100a-Guitare-Bends.xml'
+        );
+    });
```

Comment:
added in [17a1103](https://github.com/CoderLine/alphaTab/pull/1462/commits/17a110328aee4a62b76eba65516d85bb2bffd68a) and fixed in [9a94544](https://github.com/CoderLine/alphaTab/pull/1462/commits/9a94544962bd849aad31017062e43ec0953d0f9e). I removed BendType checks because the way bend types are determined doesn't seem to work well with the way musicxml represents bends.